ruby/alias_manager.rb

```diff
@@ -0,0 +1,67 @@
+# write a method that takes a spy's real name (e.g., "Felicia Torres")
+# and creates a fake name with it by doing the following:
+# Swapping the first and last name
+
+# Change all of the vowels to the next vowel in 'aeiou'
+def change_vowels(vowel)
+  vowels = 'aeioua'
+  new_vowel = vowels.index(vowel)
+  next_vowel = new_vowel + 1
+  swapped_vowel = vowels[next_vowel]
+end
+# p change_vowels("a")
+# p change_vowels("e")
+# p change_vowels("i")
+# p change_vowels("o")
+# p change_vowels("u")
+
+# Change all consonants to the next consonant exculding all vowels
+def change_consonant(consonant)
+  consonants = 'bcdfghjklmnpqrstvwxyzb'
```

Comment:
'b' the beginning and end is clever but error prone.

just use conditionals for the edge case